charts/drax/templates/secret-grafana-slack-contactpoints.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if $.Values.global.alerting.enabled -}}
+{{- if eq $.Values.global.alerting.slackToken "" -}}
+  {{- fail "Error: global.alerting.slackToken must be defined." -}}
+{{- end -}}
+{{- end -}}
+
+{{- $secretName := printf "%s-slack-auth" $.Release.Name -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ $secretName }}
+  annotations:
+    "helm.sh/resource-policy": keep
+type: Opaque
+data:
+  {{- if $.Values.global.alerting.enabled -}}
```

Comment:
Is this necessary? if alerting is not enabled, then no secret should be created which is what happens based on the top condition. So we will never end up in the else bellow

Reply:
It would be ideal indeed but we refer to it in the grafana environment variable so if it is not created the deployment fails